### ISSUE TITLE
Torch.optim Errata

### ISSUE THREAD

**Issue by richardachen** [pytorch/pytorch#91080]
### 📚 The doc issue

In “To use torch.optim you have to construct an optimizer object, that will hold the current state and will update the parameters based on the computed gradients”, change “object, that” to “object that”.

In “can be also easily integrated”, change “be also” to “also be”.

cc @svekars @carljparker

**Comment by albanD** [comment]
Thanks for the suggestion, feel free to send a PR fixing these!
